web/docs/Backend Starters/Decoupled WordPress/configuring-build-hooks.md

```diff
@@ -0,0 +1,57 @@
+---
+id: 'configuring-build-hooks'
+title: 'Configuring Build Hooks'
+slug: '/backend-starters/decoupled-wordpress/configuring-build-hooks'
+sidebar_position: 2
+---
+
+For a decoupled technology stack, build hooks bring back a way to kick off and
+provide updates to a user’s site that is often lost when moving away from
+tightly coupled ecosystems. This feature allow user's to quickly trigger new
+builds of a site when updates are made on their blog, CMS or other data source.
+With build hooks, when a configured source of data is updated a deployment is
+triggered for it's corresponding front-end site.
+
+## Install and Activate WP Webhooks Plugin
```

Comment:
I think this section could flow a bit better if it falls under a single heading like "Create A Buildhook" or something like that
```
## Create Your First Buildhook
### Install And Activate WP Webhooks Plugin
### Trigger A Build On Content Update And Publish
### Trigger A Build For Multiple Frontend Environments
```